cabal-install/src/Distribution/Client/Utils/Parsec.hs

```diff
@@ -103,3 +115,38 @@ advance n z@(Zipper xs ys)
   | otherwise = case ys of
       [] -> z
       (y : ys') -> advance (n - 1) $ Zipper (y : xs) ys'
+
+-- | Like 'List', but for 'NubList'.
+--
+-- @since 3.2.0.0
```

Comment:
Was this taken from another part of the codebase? It can’t be `since 3.2.0.0` otherwise, right?

Reply:
whoops. yes, good catch. It stems from `Distribution.FieldGrammar.Newtypes`, forgot to delete this.